### PR TITLE
Spec branch

### PR DESCRIPTION
**Pull request by emdurr**
**Description**

Jest Implementation

### CODE REVIEW

File: client/config/setup_tests.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+import 'jasmine-ajax';
```
Comment (BenjaminCall) [inline client/config/setup_tests.js]: Need this?

Comment (emdurr) [inline client/config/setup_tests.js]: Yes

File: client/js/_author/components/assessments/question_types/question_common/save_option_button.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// import React          from 'react';
+var React = require('react');
+// import SaveOption     from './save_option_button';
+
+test('renders an onClick button', () => {
+  // const saveButton = shallow(
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/question_common/save_option_button.spec.jsx]: ???

File: client/js/_author/components/assessments/question_types/question_common/save_option_button.spec.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+// import React          from 'react';
+var React = require('react');
+// import SaveOption     from './save_option_button';
+
+test('renders an onClick button', () => {
```
Comment (BenHSmith13) [inline client/js/_author/components/assessments/question_types/question_common/save_option_button.spec.jsx]: test or describe? Lets start a holy war over this

File: package.json [hunk 4]
```diff
@@ -29,5 +29,16 @@
   "dependencies": {
     "del": "^2.2.2"
   },
-  "devDependencies": {}
+  "devDependencies": {
```
Comment (BenHSmith13) [inline package.json]: can we delete this?

**Pull request merged**